src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -363,6 +365,11 @@ def search_subroutine(nodelist, searchname):
         try:
             subroutines = first_type_match(mod_content,
                                            Fortran2003.Module_Subprogram_Part)
+            # TODO remove once fparser/#102 is done
+            module_ast.content[0]._parent = module_ast
```

Comment:
pylint complains about accessing _parent here and at a number of locations in the new code in this file.

Reply:
Oh yes, fixed.